### ISSUE TITLE
Feature Request when updating a password

### ISSUE THREAD

**Issue by jeff9315** [mihaifm/HIBPOfflineCheck#10]
It would be Super Nice to display the number of pwned entries found when editing a password field ... and displaying it before you save the item. I know it's displayed in the column, but it's too easy to miss that. Maybe something that couldn't be quite so easily ignored.  A notice that stays there for 5 seconds or is in red or something that needs to be clicked to be dismissed. Actually, the more obnoxious, the better :-)

Thanks AGAIN ... Jeff

**Comment by mihaifm** [comment]
Ok, a messagebox wouldn't be too hard to add, but it needs to be enabled via a config dialogue, so I need to add that enhancement as well.

**Comment by mihaifm** [comment]
Feature request implemented: 146106f

`Tools -> HIBP Offline Check... -> Display warning after editing insecure passwords`

Can you help verifying it?

**Comment by jeff9315** [comment]
I was able to verify it. It works. It took me a minute to realize I had to enable that is in options box first. 

It showed up after hitting Enter (which is OK). The plugin (KeePassHIBP.plgx) displays a brief message during entry but is far too brief. Maybe that'll be a help to determine if there's a better way to display it.

Perhaps showing the Password Count (as it shows in the column) found would be another nice-to have feature.
